solara/lab/components/chat.py

```diff
@@ -0,0 +1,198 @@
+import uuid
+from typing import Callable, Dict, List, Optional, Union
+
+from typing_extensions import Literal
+
+import solara
+from solara.components.input import use_change
+
+
+@solara.component
+def ChatBox(
+    children: List[solara.Element] = [],
+    style: Optional[Union[str, Dict[str, str]]] = None,
```

Comment:
```suggestion
    style: Optional[Union[str, Dict[str, str]]] = {"flex-grow": "1", "flex-direction": "column-reverse", "overflow-y": "auto"},
```

Reply:
I would do this one the same way as the others (conditionally). I think users might find it confusing if they aren't familiar with CSS and want to change something like `{"background-color": "green"}`, and that makes the container not work. Let me know what you think.

Reply:
True, lets keep it this way.